src/Tribe/Importer/Column_Mapper.php

```diff
@@ -37,10 +37,17 @@ public function set_defaults( $defaults ) {
 		$this->defaults = $defaults;
 	}
 
+	/**
+	 * Render a select dropdown at the top of a column for column mapping.
+	 *
+	 * @param int $index The index of the column. Starts with 0.
+	 *
+	 * @return string The HTML markup of the select box.
+	 */
```

Comment:
This needs a `@since TBD` line.

Reply:
```suggestion
	/**
	 * Render a select dropdown at the top of a column for column mapping.
	 *
	 * @since TBD
	 *
	 * @param int $index The index of the column. Starts with 0.
	 *
	 * @return string The HTML markup of the select box.
	 */
```

Reply:
Can I add @since 1.0.0 ??? 😆

Reply:
Added since 3.10.0